lib/matplotlib/style/__init__.py

```diff
@@ -1,3 +1,6 @@
 from __future__ import absolute_import
 
 from .core import use, context, available, library, reload_library
+from matplotlib import rcParams
+if rcParams['style']:
+    use(rcParams['style'])
```

Comment:
After a small test, I confirmed something I already noticed before, which is that a style call from the matplotlibrc *will* be applied even without calling the `matplotlib.style` library. I guess that on init, matplotlib initializes also the style library, or giving the style parameter in the rc will call the style library... I'm not sure about which it is